client/tokenized-express-checkout/index.js

```diff
@@ -535,40 +515,7 @@ jQuery( ( $ ) => {
 		 * Initialize event handlers and UI state
 		 */
 		init: () => {
-			if (
-				getExpressCheckoutData( 'button_context' ) === 'pay_for_order'
-			) {
-				if ( ! window.wcpayECEPayForOrderParams ) {
-					return;
-				}
-
-				const {
-					total: { amount: total },
-					displayItems,
-					order,
-				} = wcpayECEPayForOrderParams;
-
-				if ( total === 0 ) {
-					expressCheckoutButtonUi.hideContainer();
-					expressCheckoutButtonUi.getButtonSeparator().hide();
-					return;
-				}
-
-				wcpayECE.startExpressCheckoutElement( {
-					mode: 'payment',
-					total,
-					currency: getExpressCheckoutData( 'checkout' )
-						?.currency_code,
-					requestShipping: false,
-					requestPhone:
-						getExpressCheckoutData( 'checkout' )
-							?.needs_payer_phone ?? false,
-					displayItems,
-					order,
-				} );
-			} else if (
```

Comment:
The pay-for-order & cart/checkout pages are so similar, that the same implementation can be used for both.